chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/impl/TestingLabManagerImpl.java

```diff
@@ -22,7 +22,6 @@
 import gov.healthit.chpl.manager.ActivityManager;
 import gov.healthit.chpl.manager.TestingLabManager;
 import gov.healthit.chpl.manager.UserPermissionsManager;
-import gov.healthit.chpl.util.AuthUtil;
```

Comment:
If you are going to be editing this class, please delete the Impl code and move it to the non-Impl package/naming convention